.pipelines/templates/build-conf-test-image.yaml

```diff
@@ -1,6 +1,23 @@
+parameters: 
+  - name: imageScan
+    type: boolean
+    default: false
+  - name: imageName
+    type: string
+    default: osm-arc-conformance
+  - name: imageTag
+    type: string
+    default: test
+
 steps:
   - bash: |
-      docker build $(plugin.path)/. -t $(CONF_REPOSITORY_NAME):$(plugin.version)
+      if [[ "${{ parameters.imageScan }}" == "True" ]]; then
```

Comment:
Is the `True` setting already being used elsewhere? It can lead to issues taking this approach (e.g. using `true` would not evaluate like you would think here). Typically to get around this shell scripts usually use 1 or 0. I'm ok with using "true" but let's test it by lower-casing it and then compare to `true` so that it is effectively case-insensitive. In bash you can do that with this parameter expansion: `${MY_STRING@L}` (this lower-cases it, `U` would uppercase it).

Reply:
We have been using "True" in other places as well. Ex: https://github.com/Azure/osm-azure/blob/main/.pipelines/templates/publish-helm-chart.yaml#L26. I remember when I was first using boolean parameters "true" didn't work for some reason, even when I was passing in "true" or "false" to the template, but I can try lower casing it and comparing it to "true"

Reply:
@trstringer I can do this in a follow up PR as the change will also touch the publish-helm-chart yaml file